src/licenseGen/Program.cs

```diff
@@ -114,13 +115,20 @@ bool verifyTopOptions()
                         if ( checkEmail(buff) )   email = buff;
                     }
 
+                    while (storage == 0)
+                    {
+                        WriteLineOver("Extra storage space for the user " + name + ". (max.: " + short.MaxValue + ") [storage]");

```

Comment:
@kerenon Would you be able to update this slightly so that the default is `short.MaxValue`?

Adding something like "default: max" to the text prompt and ensuring that the program sets the value to `short.MaxValue` if/when a user simply hits enter at the prompt.